pennylane/templates/tensornetworks/mps.py

```diff
@@ -113,39 +125,65 @@ def circuit(template_weights):
         2: ───────────────╰X──RY(-0.30)─╭●──RY(0.10)──┤
         3: ─────────────────────────────╰X──RY(-0.30)─┤  <Z>
 
-    """
+        MPS can also be used with an ``offset`` argument that shifts the positioning the subsequent blocks relative to the default ``n_block_wires/2``.
+
+        .. code-block:: python
+
+            import pennylane as qml
+            import numpy as np
+
+            def block(weights, wires):
```

Comment:
```suggestion
            def block(wires):
```
Having the weights causes an error unless we add weights in the `qml.MPS` call

Reply:
fixed.